proto/provenance/msgfees/v1/proposals.proto

```diff
@@ -11,8 +11,13 @@ option java_package        = "io.provenance.msgfees.v1";
 option java_multiple_files = true;
 
 // AddMsgFeeProposal defines a governance proposal to add additional msg based fee
+// Deprecated: This message is no longer usable. It is only still included for
+// backwards compatibility (e.g. looking up old governance proposals).
+// It is replaced by providing a MsgAddMsgFeeProposalRequest in a governance proposal.
 message AddMsgFeeProposal {
-  option (gogoproto.equal) = true;
+  option deprecated                   = true;
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_stringer) = true;
```

Comment:
Both of these default to `true`, so they can be omitted.

Reply:
Hmmm. I was wrong about `gogoproto.equal`. I guess it does default to `false`. No biggie though. We shouldn't need an equals method on these, so all good.